pages/game/[slug].vue

```diff
@@ -0,0 +1,30 @@
+<script setup lang="ts">
+import { useRoute, createError } from '#imports'
+import Loader from 'blocks/Loader/Loader.vue'
+import { useGetLeaderboardBySlug } from '~/composables/api'
+import LeaderboardInfo from '~/components/blocks/LeaderboardInfo/LeaderboardInfo.vue'
+const {
+  params: { slug },
+} = useRoute()
+
+const {
+  loading,
+  error: leaderboardError,
+  data,
+} = await useGetLeaderboardBySlug(slug as string)
+
+if (leaderboardError?.status === 404) {
+  throw createError({
+    statusCode: 404,
+    statusMessage: 'Page Not Found',
```

Comment:
Maybe this should say "Board Not Found" instead?

Reply:
I can get behind this